libkbfs/root_metadata.go

```diff
@@ -280,6 +331,242 @@ func (md *RootMetadata) swapCachedBlockChanges() {
 	}
 }
 
+// GetTLFCryptKeyParams wraps the respective method of the underlying BareRootMetadata for convenience.
```

Comment:
What's the reason for unembedding again? Currently it doesn't seem to buy much, since we have all these delegate functions. Will some of these be omitted in the future?